brave/brave-ios
# Fix #3491: Add New Menu UI

## Summary of Changes


This pull request fixes #3491 

This also changes private mode theming to only affect the main browser toolbars

## Submitter Checklist:


## Test Plan:

- Verify all menu options from old design are available in the new design and function correctly
- Verify theming works as needed
- Verify VPN button toggles correctly, shows loading accordingly while connecting, and shows the correct on/off state when opening the menu
- **Note**:  #3672 moves the share with… button to the top, which will be inconsistent with screenshots below

## Screenshots:

| Old (NTP) | New (NTP) | 
| --- | --- | 
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-06 at 11 00 48](https://user-images.githubusercontent.com/529104/117320577-5ddd3580-ae5a-11eb-8290-38f4eee8823d.png) | ![new menu](https://user-images.githubusercontent.com/529104/117319657-84e73780-ae59-11eb-90c6-acbe28dd763e.png) |

| Old (Website) | New (Website) | 
| --- | --- | 
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-06 at 11 00 52](https://user-images.githubusercontent.com/529104/117320811-95e47880-ae5a-11eb-92d9-2a80cc0e3e02.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-06 at 10 54 18](https://user-images.githubusercontent.com/529104/117320144-f32bfa00-ae59-11eb-92d7-6c029be3e999.png) |

| Old (Website) | New (Website) Expanded | 
| --- | --- | 
| ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-06 at 11 00 52](https://user-images.githubusercontent.com/529104/117320811-95e47880-ae5a-11eb-92d9-2a80cc0e3e02.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-05-06 at 10 54 20](https://user-images.githubusercontent.com/529104/117320161-f58e5400-ae59-11eb-816e-71d40fc13b94.png)  |

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
